hazelcast/src/test/java/com/hazelcast/concurrent/lock/LockBounceMemberTest.java

```diff
@@ -0,0 +1,136 @@
+package com.hazelcast.concurrent.lock;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.ILock;
+import com.hazelcast.test.AssertTask;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.annotation.SlowTest;
+import com.hazelcast.test.bounce.BounceMemberRule;
+import com.hazelcast.test.jitter.JitterRule;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.HashSet;
+import java.util.Random;
+
+import static com.hazelcast.test.HazelcastTestSupport.assertTrueEventually;
+import static com.hazelcast.test.HazelcastTestSupport.generateKeyNotOwnedBy;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+/**
+ * Runs a cluster of {@value MEMBER_COUNT + 1} and calls lease lock from {@value DRIVER_COUNT} drivers. During the test bounces
+ * {@value MEMBER_COUNT} cluster members (shuts down and restarts). The remaining 1 member is stable (never bounced) but no
+ * locks are owned by this member.
+ * Asserts that all locks are unlocked eventually.
+ */
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(SlowTest.class)
+public class LockBounceMemberTest {
```

Comment:
Can you rename the test to make it clear `lock with lease` is being tested? Something like `LockWithLeaseBounceMemberTest`, `LeaseLockBounceMemberTest` ... etc